packages/main/src/components/protected/topbar/styles/Topbar.module.css

```diff
@@ -310,10 +310,10 @@
   font-weight: 400;
   line-height: 19.5px;
   cursor: pointer;
-  margin: 0.5rem 0;
+  /* margin: 0.5rem 0; */
```

Comment:
Instead of commenting out the changes, if the code is not in use\ remove it entirely from the file